src/sdg/producten/forms.py

```diff
@@ -168,6 +168,11 @@ def _help_text(self, field):
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+        if self.instance.product_valt_onder:
+            selected_doelgroep = self.instance.product_valt_onder.generiek_product
```

Comment:
Naming vars makes things more clear.

```suggestion
            selected_product = self.instance.product_valt_onder.generiek_product
```